frontend/test/metabase/scenarios/admin/databases/add-external.cy.spec.js

```diff
@@ -1,129 +1,133 @@
 import { restore, typeAndBlurUsingLabel } from "__support__/e2e/helpers";
 
-describe("admin > database > add > external databases", () => {
-  beforeEach(() => {
-    restore();
-    cy.signInAsAdmin();
-
-    cy.intercept("POST", "/api/database").as("createDatabase");
-  });
-
-  it("should add Postgres database and redirect to listing (metabase#17450)", () => {
-    cy.visit("/admin/databases/create");
-    cy.contains("Database type").closest(".Form-field").find("a").click();
-    cy.contains("PostgreSQL").click({ force: true });
-
-    cy.findByText("Show advanced options").click();
-    cy.contains("Additional JDBC connection string options");
-    // Reproduces metabase#17450
-    cy.findByLabelText("Choose when syncs and scans happen")
-      .click()
-      .should("have.attr", "aria-checked", "true");
-
-    isSyncOptionSelected("Never, I'll do this manually if I need to");
-
-    typeAndBlurUsingLabel("Display name", "QA Postgres12");
-    typeAndBlurUsingLabel("Host", "localhost");
-    typeAndBlurUsingLabel("Port", "5432");
-    typeAndBlurUsingLabel("Database name", "sample");
-    typeAndBlurUsingLabel("Username", "metabase");
-    typeAndBlurUsingLabel("Password", "metasample123");
-
-    cy.button("Save").should("not.be.disabled").click();
-
-    cy.wait("@createDatabase");
-
-    cy.url().should("match", /\/admin\/databases\?created=true$/);
+describe(
+  "admin > database > add > external databases",
+  { tags: "@external" },
```

Comment:
Unfortunately, the linter made a mess out of this diff, but this line here is the only changed part.